discourse_migration/discourse-categories.md

```diff
@@ -0,0 +1,75 @@
+# Proposed Discourse Categories
+
+* Announcements
+
+* Releases
+
+* Clang Compiler & Tools
+  - Using Clang
+  - Building Clang
+  - clangd
+
+* Subprojects
+  - lld
+  - Flang
+  - lldb
+  - Static Analyzer
+
+* Code Generation
+  - AArch64
```

Comment:
I'd suggest "Common Infra" or something similar as a subcategory.

Reply:
Can we use the top-level category for that?

Reply:
My thought is that "Code Generation / Common Infra" would be things like SelectionDAG, Global ISel, TargetRegisterInfo, etc.  All Code generation related things.